app/models/action.js

```diff
@@ -0,0 +1,44 @@
+import DS from 'ember-data';
+import { attr, hasMany, belongsTo } from '@ember-decorators/data';
+
+const { Model } = DS;
+
+export default class ActionModel extends Model {
+// DB table: dcp_projectaction
+
+  // One Project to Many Actions
+  @belongsTo('project') project;
+
+  // One Hearing to One Action
+  @belongsTo('hearing') hearing;
+
+  // Many Recommendations to One Action
+  @hasMany('recommendation') recommendations;
```

Comment:
This should be the other way around. One recommendation can have many actions. Participants have the option of either A) submitting multiple recommendations, one recommendation for each action, or B) submitting a single recommendation for all actions at once.

Reply:
So are these many-to–many? Many recommendations can be related to many actions?

Reply:
Ahh okay gotcha --will make the changes accordingly

Reply:
ohhh yes it's many to many!